vaadin-grid-pro-flow-integration-tests/src/test/java/com/vaadin/flow/component/gridpro/test/BasicIT.java

```diff
@@ -26,6 +27,16 @@ public void editColumnsAdded() {
         Assert.assertEquals(columns.size(), 3);
     }
 
+    @Test
+    public void columnIsRenderedInBeanGrid() {
+        GridTHTDElement cell = beanGrid.getCell(0, 0);
+        Assert.assertEquals("Person 1", cell.getInnerHTML());
+        // Entering edit mode with double click
+        executeScript("var cellContent = arguments[0].firstElementChild.assignedNodes()[0];" +
+                "cellContent.dispatchEvent(new CustomEvent('dblclick', {composed: true, bubbles: true}));", cell);
+        Assert.assertTrue(cell.innerHTMLContains("vaadin-grid-pro-edit-text-field"));
```

Comment:
Duplicating this complex logic is bad for maintenance.
Create another overload of `AssertCellEnterEditModeOnDoubleClick`, which takes the grid element as the parameter. In the existing overload, call that one with the default grid.

Reply:
Updated.